com.unity.netcode.gameobjects/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -1017,12 +1018,13 @@ private void InjectWriteAndCallBlocks(MethodDefinition methodDefinition, CustomA
                     }
                     else
                     {
-                        m_Diagnostics.AddError(methodDefinition, $"Don't know how to serialize {paramType.Name} - implement {nameof(INetworkSerializable)} or add an extension method for {nameof(FastBufferWriter)}.{k_WriteValueMethodName} to define serialization.");
```

Comment:
again, why are we converting from nameof to non-nameof hardcoded strings?

Reply:
It was an accident. I accidentally did my changes in the wrong project, copy-pasted into testproject without realizing what I was in was out of date, and foolishly didn't look at the diff before submitting. Rollback incoming.